api/src/models/bebida.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const { ObjectId } = mongoose.Schema.Types;
+const { Schema } = mongoose;
+
+/** @description Representacion de un Bebida de un Restaurante */
+const BebidaSchema = new Schema({
+  codigo: { type: String },
+  nombre: { type: String },
+  descripcion: { type: String },
+  ingredientes: { type: String },
+  marca: { type: ObjectId, ref: 'Marca' },
+  restaurante: { type: ObjectId, ref: 'Restaurante' },
+  nacionalidad: { type: ObjectId, ref: 'Medida' },
```

Comment:
En nacionalidad creo que seria ref: 'Pais'

Reply:
Changes implemented in commit ~> 61e239e